apm-agent-plugins/apm-rabbitmq/apm-rabbitmq-spring/src/main/java/co/elastic/apm/agent/rabbitmq/MessageBatchHelper.java

```diff
@@ -1,3 +1,21 @@
+/*
```

Comment:
It feels like all of these rabbitmq changes should move to different PR?

Reply:
No need, this about missed licensed headers. This headers is already at master branch now.